File: Examples/Node.js/SDK/manipulate/CropImageWithFormatChange.js
var fs = require('fs'),
    utils = require('../utils.js'),
    image = utils.imagingApi;
    storage = utils.storageApi;

var inputName = "sample1.png";
var inputPath = utils.getPath(__filename, inputName);

var outputName = "nodejsSDK.output.jpg";
var outputPath = utils.getPath(__filename, outputName);

var x = 10;
var y = 20;
var width = 50;
var height = 50;
var Storage = null;
var folder = null;
var save_format = 'jpg';

storage.PutCreate(inputName, null, Storage, inputPath, function (responseMessage) {
    image.GetCropImage(inputName, save_format, x, y, width, height, null, folder, Storage, function (responseMessage) {
        storage.GetDownload(inputName, null, null, function (responseMessage) {
            var writeStream = fs.createWriteStream(outputPath);
            writeStream.write(responseMessage.body);
        });
    });
});

File: Examples/Python/SDK/manipulate/CropImageWithFormatChange.py
import common

input_file = "sample1.png"
input_path = common.get_path(__file__, input_file)

output_file = "output.jpg"
output_path = common.get_path(__file__, output_file)

common.StorageApi.PutCreate(input_file, input_path)

format = "jpg"
x = 30
y = 40
width = 100
height = 100
# invoke Aspose.Imaging Cloud SDK API to perform format change operation without using cloud storage
response = common.imagingApi.GetCropImage(input_file, format, x, y, width, height)

if response.Status == 'OK':
    # download image from API response
    with open(output_path, 'wb') as f:
        for chunk in response.InputStream:
            f.write(chunk)
